third_party/blink/renderer/platform/heap/member.h

```diff
@@ -91,6 +93,69 @@ struct MemberHash
   }
 };
 
+// Replay's hashing function with Member<>-wrapped objects, using the registered 
+// pointer id.  
+template <typename T>
+struct MemberHashRecordReplayRegisteredPointerId
```

Comment:
Given that all of the actual uses of the modified hashtable uses the other variant, it might make sense to leave this hasher out for now.

I haven't actually seen `PointerId()` and friends used in this codebase yet.  They tend to be more heavily used in gecko code, but in chromium we're using the lower overhead approach of obtaining an explicit ID and passing it around with the object whose identity is being stably tracked.

Reply:
We do use RegisterPointer in a fair number of places in chromium.  Originally we used it everywhere we needed a stable ID, and changed things to the explicit ID for classes which were using PointerId a lot --- PointerId is a lot slower than just reading an integer field.